Volume/View Models/MagazineCell.swift

```diff
@@ -8,23 +8,29 @@
 
 import SDWebImageSwiftUI
 import SwiftUI
+import PDFKit
 
 /// `MagazineCell` displays the images and name of a publication a user is currently following
 struct MagazineCell: View {
 
     let magazine: Magazine
-    // let largeFont: Bool
+    let magazineUrl: URL
 
     var body: some View {
         VStack(alignment: .leading) {
-            // TODO: figure out how to obtain cover image from PDF
-            WebImage(url: URL(string: "https://picsum.photos/200")!)
-                .resizable()
-                .grayBackground()
-                .aspectRatio(contentMode: .fill)
-                .frame(width: 150, height: 220)
-                .clipped()
-                .shadow(color: Color.black.opacity(0.2), radius: 8, x: 4, y: 4)
+            if let pdfDoc = PDFDocument(url: magazineUrl) {
```

Comment:
If you're requesting a PDF from this URL one time here, and another time in the MagazineReaderView. Is there a way to pass in loaded PDFDocument objects from MagazineList and reuse them instead?